app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,8 @@
 
   <body>
     <% if signed_in? %>
-      Signed in as: <%= current_user.email %>
+      Signed in as: <%= current_user.name %>
```

Comment:
Ah ok. In that case, could you remove noface_image? Seems like it'd be cleaner to do that, rather than merging in with a file that isn't currently needed by any of the code. If it stays, other pairs might get confused about what it's for.